### ISSUE TITLE
[Feature] Freedoom patch

### ISSUE THREAD

**Issue by proydoha** [Alando1-doom/Toby-Accessibility-Mod-for-Doom#33]
Given that apparently Toby Accessibility Mod gets played with Freedoom it could be worth the effort to make a compatibility patch for it.

I believe that it should be a separate patch, not part of the main package because actor names are the same between Doom and Freedoom.

Maybe its worth creating a separate repo for various compatibility patches ( ? )

We could include new sounds for Actors-in-viewport feature: Freedoom monsters, items and weapons have their own unique names.
Not sure about decorative actors.

Map name announcement feature can be included in that patch too.

Record sounds (singular and plural) for:
- [ ] Items
- [ ] Ammo
- [ ] Weapons
- [ ] Keys
- [ ] Monsters
- [ ] Player

Record sounds for map names:
- [ ] Freedoom 1 maps
- [ ] Freedoom 2 maps

Looking at DEHACKED lump for names:

Items:
- light armor vest
- heavy armor vest
- small health pack
- large health pack
- overdrive sphere
- ultra-overdrive sphere
- stealth sphere
- hazard suit
- tactical survey map
- health bonus - same as Doom
- armor bonus - same as Doom
- night vision goggles - same as Doom
- invulnerability - same as Doom
- berserk - same as Doom

Ammo:
- ammo clip - same as Doom
- box of ammo - same as Doom
- missile
- crate of missiles
- energy recharge
- large energy recharge
- some shotgun shells - existing sounds will do
- box of shotgun shells - existing sounds will do
- backpack - same as Doom

Weapons:
- SKAG 1337
- minigun
- angle grinder
- missile launcher
- polaric energy cannon
- pump-action shotgun
- double-barrelled shotgun
- fist - same as Doom
- pistol - same as Doom

Keys:
- blue passcard
- yellow passcard
- red passcard
- blue skeleton key
- yellow skeleton key
- red skeleton key

Monsters:
- zombie
- shotgun zombie
- minigun zombie
- serpentipede
- flesh worm
- deadflare
- trilobite
- pain bringer
- pain lord
- technospider
- summoner
- dark soldier
- combat slug
- necromancer
- large technospider
- assault tripod

Player:
- savior of humanity

Freedoom 1 map names
Episode 1:
- E1M1: Outer Prison
- E1M2: Communications Center
- E1M3: Waste Disposal
- E1M4: Supply Depot
- E1M5: Main Control
- E1M6: Training Facility
- E1M7: Transportation Bay
- E1M8: Outpost Quarry
- E1M9: Armory

Episode 2:
- E2M1: Ruins
- E2M2: Power Plant
- E2M3: Archaelogy Site
- E2M4: Sample Holding Site
- E2M5: Fortress 31
- E2M6: Trepidation Site
- E2M7: Quarantine Vessel
- E2M8: Containment Cell
- E2M9: Corruption of Man

Episode 3:
- E3M1: Land of the Lost
- E3M2: Infernal Caverns
- E3M3: Derelict Temple
- E3M4: Sacrificial Bastion
- E3M5: Oblation Temple
- E3M6: Igneous Intrusion
- E3M7: No Regrets
- E3M8: Ancient Lair
- E3M9: Acquainted With Grief

Episode 4:
- E4M1: Maintenance Area
- E4M2: Research Complex
- E4M3: Central Computing
- E4M4: Hydroponic Facility
- E4M5: Engineering Station
- E4M6: Command Center
- E4M7: Waste Treatment
- E4M8: Launch Bay
- E4M9: Operations

Freedoom 2 map names:
- MAP01: Hydroelectric Plant
- MAP02: Filtration Tunnels
- MAP03: Crude Processing Center
- MAP04: Containment Bay
- MAP05: Sludge Burrow
- MAP06: Gamma Labs
- MAP07: Outer Storage Warehouse
- MAP08: Astronomy Complex
- MAP09: Datacenter
- MAP10: Deadly Outlands
- MAP11: Infinite Plain
- MAP12: Railroads
- MAP13: Station Earth
- MAP14: Nuclear Zone
- MAP15: Hostile Takeover
- MAP16: Urban Jungle
- MAP17: City Capitol
- MAP18: Aquatics Lab
- MAP19: Sewage Control
- MAP20: Blood Ember Fortress
- MAP21: Under Realm
- MAP22: Remanasu
- MAP23: Underground Facility
- MAP24: Tertiary Loading Bay
- MAP25: Red Works
- MAP26: Dark Depths
- MAP27: Warped Elementality
- MAP28: Grim Redoubt
- MAP29: Last Stand
- MAP30: Jaws of Defeat
- MAP31: Be Quiet
- MAP32: Not Sure

**Comment by Alando1-doom** [comment]
We can do it. At least some of the pre-existing Toby Sounds can be used. Once version 7.0 is done, I'll be happy to do some voiceover work for Freedoom :)
